frame/nfts/src/lib.rs

```diff
@@ -499,8 +499,10 @@ pub mod pallet {
 		WrongOwner,
 		/// Invalid witness data given.
 		BadWitness,
-		/// The item ID is already taken.
-		InUse,
+		/// Collection ID is already taken.
+		CollectionIdInUse,
+		/// Items within that collection are non-transferable.
+		ItemsNotTransferable,
 		/// The item or collection is frozen.
```

Comment:
would be good to say what 'frozen' means here. Be good to state the difference between frozen and locked.

Reply:
Done